crescent/ext/tasks/task.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from asyncio import ensure_future, get_event_loop
```

Comment:
Unless we do it this way elsewhere, `import asyncio` and `asyncio.get_event_loop()` feels cleaner.

Reply:
Every single other non-example file uses from imports